ddtrace/internal/writer.py

```diff
@@ -569,7 +569,13 @@ def flush_queue(self, raise_exc=False):
                 if raise_exc:
                     e.reraise()
                 else:
-                    log.error("failed to send traces to Datadog Agent at %s", self._agent_endpoint, exc_info=True)
+                    log.error(
+                        "failed to send %d traces to Datadog Agent at %s after %d retries (%s)",
```

Comment:
should we be more explicit that the traces are dropped?

Reply:
```suggestion
                        "failed to send, dropping %d traces to Datadog Agent at %s after %d retries (%s)",
```
?

Reply:
yeah, I like it